examples/__testfixtures__/styled-components-to-ucl/basic.output.ts

```diff
@@ -45,11 +45,16 @@ export const Tokens = Box.withConfig({
   borderWidth: 2,
   borderColor: Styles.color.primary,
   borderStyle: 'solid',
+
+  // @ts-ignore web only attribute
```

Comment:
I'm open to changing the message here?

Reply:
are we sure it wont' crash or something?

Reply:
actually `header` is fine, `section` and `nav` are no
![Screen Shot 2022-03-17 at 4 16 22 PM](https://user-images.githubusercontent.com/93778928/158888180-8e43ea3d-9457-48e7-bfba-787f52f6d364.png)
t